src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationSideTests.java

```diff
@@ -239,10 +234,9 @@ public void MercuryNavigationSideTest_003_ValidSuggestionRedirect() {
     );
   }
 
-  // NST04
   // TODO: check article titles instead of url, check that receive at least 3 different articles in 10 attempts
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)